src/components/FwbCard/composables/useCardClasses.ts

```
@@ -13,17 +14,20 @@
+    } else if (props.variant.value === 'default') {
+      computedClasses = 'block max-w-sm rounded-lg border border-gray-200 shadow-md dark:border-gray-700'
+    } else if (props.variant.value === 'horizontal') {
+      computedClasses = 'flex flex-col items-center rounded-lg border shadow-md md:flex-row md:max-w-xl dark:border-gray-700'
     }
 
-    if (props.variant.value === 'default') {
-      computedClasses = 'block max-w-sm bg-white rounded-lg border border-gray-200 shadow-md dark:bg-gray-800 dark:border-gray-700 '
-    } else if (props.variant.value === 'horizontal') {
-      computedClasses = 'flex flex-col items-center bg-white rounded-lg border shadow-md md:flex-row md:max-w-xl dark:border-gray-700 dark:bg-gray-800'
+    if (!props.class?.value || (!props.class.value.includes('bg-'))) {
+      computedClasses += ' bg-white dark:bg-gray-800';
```

ESLint

[error] Extra semicolon.

(@stylistic/semi)